script/Ciel/Day 13/13_01_02_CIEL13_6.txt

```diff
@@ -216,27 +216,27 @@ Without straining or relaxing my attention, I sweep backward as if I were fallin
 [sha:096dfb8e006e7e3396668c4a97c901d9c5a9adc9]{
 -- Page 120, Offset 35598.
 -- ブリッジに近い姿勢で、飛びこんできたノエルごと凶器をやり過ごす。
-With a bridge-like stance, I let Noel fly past together with her deadly weapon.
+Assuming a posture akin to a bridge, I let Noel slip right over me.
```

Comment:
what does this even mean, what

Reply:
![image](https://user-images.githubusercontent.com/97076550/207817795-896d9229-8ab9-4473-8839-3f40ca85d3fd.png)
i assumed it was something like this but yeah it is kinda stupid